types/dockerode/index.d.ts

```diff
@@ -442,6 +442,11 @@ declare namespace Dockerode {
         abortSignal?: AbortSignal;
     }
 
+    interface NetworkConnectOptions {
+        Container: string;
```

Comment:
Is this property required? Since this is a popular package, I'm worried that this change could break users of those types, since before you could have a call like `network.connect({}, () => {...})` type-check, whereas now you can't.

Reply:
The function attaches a container to a network, not giving a container ID makes the function not useful but its entirely possible people used it this way. I will make container optional.

Reply:
Edited the PR